src/frontend/package.json

```diff
@@ -16,9 +16,10 @@
     "type-check": "tsc --noEmit",
     "lint": "concurrently \"node_modules/.bin/eslint **/*.{ts,tsx} --quiet --fix\" \"node_modules/.bin/stylelint --fix '**/*.{js,ts,tsx,css}'\"  \"npm run type-check\"",
     "lint-ci": "concurrently \"node_modules/.bin/eslint **/*.{ts,tsx}\" \"node_modules/.bin/stylelint '**/*.{js,ts,tsx,css}'\"  \"npm run type-check\"",
+    "lint-e2e": "concurrently \"node_modules/.bin/eslint --fix e2e/**/*.ts\" \"node_modules/.bin/stylelint --fix 'e2e/**/*.ts'\"  \"npm run type-check\"",
     "pl:staging": "NODE_ENV=staging npx playwright test --headed -c ./e2e/setup/staging.config.ts",
     "pl:local": "NODE_ENV=local npx playwright test --headed -c ./e2e/setup/staging.config.ts",
-    "pl:staging:headless": "npx playwright test --headless -c ./e2e/setup/staging.config.ts",
+    "pl:staging:headless": "npx playwright test --headed -c ./e2e/setup/staging.config.ts",
```

Comment:
did you mean to change this?

Reply:
its headed now yup

Reply:
@luisczi this should change. Headed is already defined in line 20.